content/en/serverless/step_functions/merge-step-functions-lambda.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Merge Step Functions and Lambda Traces
+further_reading:
+    - link: '/serverless/step_functions/installation'
+      tag: 'Documentation'
+      text: 'Install Serverless Monitoring for AWS Step Functions'
+---
+
+This page describes how to merge your AWS Step Functions traces with related AWS Lambda traces or nested Step Functions traces. These instructions assume that you have already instrumented these [AWS Step Functions][1] and [Lambda functions][2] to send traces to Datadog.
```

Comment:
I would change the links to be more like

> These instructions assume that you have already [instrumented your AWS Step Functions][1] and [instrumented your Lambda functions][2] to send traces to Datadog

OR I would move the links to the first sentence.

But what you have is also fine :)